constph.py

```diff
@@ -573,14 +618,14 @@ def setTitrationState(self, titration_group_index, titration_state_index, contex
             # TODO: Handle Custom forces, looking for "charge" and "chargeProd".
             for (charge_index, atom_index) in enumerate(atom_indices):
                 if force_classname == 'NonbondedForce':
-                    [charge, sigma, epsilon] = force.getParticleParameters(atom_index)
-                    if debug: print (" modifying NonbondedForce atom %d : (charge, sigma, epsilon) : (%s, %s, %s) -> (%s, %s, %s)" % (
-                    atom_index, str(charge), str(sigma), str(epsilon), str(charges[charge_index]), str(sigma), str(epsilon)))
+                    [charge, sigma, epsilon] = map(strip_in_unit_system, force.getParticleParameters(atom_index))
```

Comment:
Don't we want to cache `[charge, sigma, epsilon]` to avoid the unit conversion overhead?  Ideally, we want to avoid calling `getParticleParameters`, which is also slow.